docs/tutorials/basic_networking.md

```diff
@@ -0,0 +1,285 @@
+![PODMAN logo](../../logo/podman-logo-source.svg)
+
+
+# Basic Networking Guide for Podman
+
+
+It seems once people master the basics of containers, networking is one of the first
+aspects they begin experimenting with.  And in regards to networking, it takes very
+little experimentation before ending up on the deep end of the pool.  The following
+guide shows the most common network setups for Podman rootfull and rootless containers.
+Each setup is supported with an example.
+
+
+## Differences between rootfull and rootless container networking
+
+One of the guiding factors on networking for containers with Podman is going to be
+whether or not the container is run by a root user or not.  This is because unprivileged
+users cannot create networking interfaces on the host.  Therefore, with rootfull
+containers, the default networking mode is to use the Container Network Interface
+(CNI) plugins and specifically the bridge plugin.  For rootless, the default network
+mode is slirp4netns. Because of the limited privileges, slirp4netns lacks some of
+the features of CNI networking; for example, slirp4netns cannot give containers a
+routable IP address.
+
+## Firewalls
+
+The role of a firewall will not impact the setup and configuration of networking,
+but it will impact traffic on those networks.  The most obvious is inbound network
+traffic to the container host, which is being passed onto containers usually with
+port mapping.  Depending on the firewall implementation, we have observed firewall
+ports being opened automatically due to running a container with a port mapping (for
+example).  If container traffic does not seem to work properly, check the firewall
+and allow traffic on ports the container is using.
```

Comment:
A common problem is that reloading the firewall deletes the cni iptables rules resulting in a loss of network connectivity for rootful containers. Podman v3 provides the podman network reload command to restore this without having to restart the container.
I think this can be added here.